# 
# @brief    : 
# @time     : 2020/09/18
# @auhtor   : liuxu <liuxu.ccc@gmail.com> 
# 

#Information about component grabbing

std_msgs/Header header

uint64 timestamp   #timestamp
float64  x
float64  y